static/js/settings_org.js

```diff
@@ -583,10 +608,10 @@ function _set_up() {
         return opts;
     }
 
-    $(".organization").on("click", ".subsection-header .subsection-changes-save button", function (e) {
+    $(".organization").on("click", ".subsection-header .subsection-changes-save .button", function (e) {
         e.preventDefault();
         e.stopPropagation();
-        var save_button = $(e.target);
+        var save_button = $(e.currentTarget);
```

Comment:
What's the story with this change?

Reply:
I guess with `currentTarget` we are sure that the save button will refer to `.subsection-header .subsection-changes-save .button`, but here I can't find any solid reason to prefer this over `target`.

Reply:
currentTarget is used because with the changes the .button container element has both a icon span and a text span. Using target would refer to these elements instead of the .button container element.

Reply:
OK, makes sense.